src/site/content/en/learn/css/sizing/index.md

```diff
@@ -0,0 +1,442 @@
+---
+title: Sizing Units
+description:
+authors:
+  - andybell
+date: 2021-04-13
+tags:
+  - css
+---
+
+# Sizing
+
+The responsive web is a flexible medium,
```

Comment:
```suggestion
The web is a responsive medium,
```

Feel free to ignore, just felt a little repetitive bw the first and last sentence "flexible medium".